_redirects

```diff
@@ -60,5 +60,7 @@ https://openzeppelin-docs.netlify.com/* https://docs.openzeppelin.com/:splat 301
 /openzeppelin/gsn/getting-started.html /learn/sending-gasless-transactions 301
 
 # OpenZeppelin Contracts 3.0 deprecation redirects
-/contracts/3.x/api/crowdsale /contracts/3.x/crowdsales 301 # Redirect Crowdsale API to deprecation page
```

Comment:
I think you made a mistake in the merge here.

Reply:
What's the mistake? I just removed the comment at the end

Reply:
Since you had just added the comment I thought removing it had been a mistake. :man_shrugging:

Reply:
Ah no, I decided to just leave the comment at the top. Thanks!